src/woo_publications/metadata/update_informatie_category.py

```diff
@@ -0,0 +1,83 @@
+from io import StringIO
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management import call_command
+
+import requests
+from glom import PathAccessError, T, glom
+
+from woo_publications.metadata.constants import InformationCategoryOrigins
+from woo_publications.metadata.models import InformationCategory
+
+SPEC = {
+    "naam": T["http://www.w3.org/2004/02/skos/core#prefLabel"][0]["@value"],
+    "naam_meervoud": T[
+        "https://identifier.overheid.nl/tooi/def/ont/prefLabelVoorGroepen"
+    ][0]["@value"],
+    "definitie": T["http://www.w3.org/2004/02/skos/core#definition"][0]["@value"],
+    "order": T["http://www.w3.org/ns/shacl#order"][0]["@value"],
+}
+
+
+class InformatieCategoryWaardenlijstError(Exception):
+    pass
+
+
+def update_informatie_category(file_path: str):
```

Comment:
```suggestion
def update_informatie_category(file_path: Path):
```
and the management command is then responsible for ensure it's a `Path` instance that's passed:

```py
file_path = options["file_path"]
if not isinstance(file_path, Path):
    file_path = Path(file_path)
...
```